torch/_torch_docs.py

```diff
@@ -5872,6 +5872,39 @@ def merge_dicts(*dicts):
     tensor([ 0,  2,  0,  1])
 """.format(**single_dim_common))
 
+add_docstr(torch.argwhere,
+           r"""
+argwhere(input) -> LongTensor
+
+Returns a tensor containing the indices of all non-zero elements of
```

Comment:
While I recognize that this documentation is copied from `torch.nonzero`'s, this might be a good opportunity to clean up the wording a little. @mruberry rightly pointed out that these two paragraphs seem to be saying the same thing over and over again and could be simplified.

Alternatively, you could file an issue about simplifying both `torch.nonzero` and `torch.argwhere` docs for later followup.

Reply:
I think I will opt filing the issue (assuming we want to/will update `torch.nonzero` to match only the `np.nonzero`)

Thanks!